source/World.cpp

```diff
@@ -174,6 +251,22 @@ namespace GTA
 
 		return gcnew Ped(handle);
 	}
+	array<Entity ^> ^World::GetAllEntities()
+	{
+		List<Entity ^> ^ents = gcnew List<Entity ^>();
+		array<int> ^entities = MemoryAccess::GetEntityHandleList();
+
+		for (int i = 0; i < entities->Length; i++)
+		{
+			if (Native::Function::Call<bool>(Native::Hash::DOES_ENTITY_EXIST, entities[i]))
+			{
+				Entity^ ent = (Entity ^)gcnew Prop(entities[i]);
```

Comment:
@JohnnyCrazy ok, sounds like a good idea. but I did it in the first place because you can't create Entity because it's abstract and Prop only has a constructor redefined and no additional properties or functions.